### ISSUE TITLE
Different Ultimate backends

### ISSUE THREAD

**Issue by danieldietsch** [FahrJo/ultimate-automizer-vscode#12]
Hey, this plugin is a great idea, thank you very much! 

Are you aware that we just recently changed our web backend? The API is somewhat compatible, but you can have a look at it at https://ultimate.sopranium.de or read the meager info we currently provide at https://github.com/ultimate-pa/ultimate/tree/dev/trunk/source/WebBackend .

**Comment by FahrJo** [comment]
Hey Daniel,
I saw yesterday that there are some changes with the backend / the API but did not yet investigate deeper. I also saw that there is a process ongoing to dockerize it? At least one of the last commits sounded like that. As you might have seen, I also started something in that way in another repo and my next plan was to provide the API inside a docker container. But if you are working on that already, I don't have to reinvent the wheel ;-)
Unfortunately I saw that changes only after I released the first version but I'll check the changes and adopt to it. 

Also there are more plans for improvements in the extension like configurable settings etc. I'll probably come back to you for that🤔
